source/geod24/LocalRest.d

```diff
@@ -2015,16 +2058,23 @@ unittest
             return this.counter;
         }
 
```

Comment:
There is still no test for stopping a timer **before** it is called, not after it is called. I suggest doing this:

```diff
diff --git a/source/geod24/LocalRest.d b/source/geod24/LocalRest.d
index b2fa160..5c5efda 100644
--- a/source/geod24/LocalRest.d
+++ b/source/geod24/LocalRest.d
@@ -2031,6 +2031,7 @@ unittest
     static interface API
     {
         public void startTimer (bool periodic);
+        public void stopTimer ();
         public ulong getCounter ();
         public void resetCounter ();
     }
@@ -2045,6 +2046,11 @@ unittest
             this.timer = setTimer(100.msecs, &callback, periodic);
         }
 
+        public override void stopTimer ()
+        {
+            this.timer.stop();
+        }
+
         public void callback ()
         {
             this.counter++;
@@ -2073,8 +2079,9 @@ unittest
     assert(node.getCounter() == 3);
     node.resetCounter();
     node.startTimer(false);
+    node.stopTimer();
     core.thread.Thread.sleep(500.msecs);
-    assert(node.getCounter() == 1);
+    assert(node.getCounter() == 0);
     node.ctrl.shutdown();
 }
 ```